conduct/COC_TEAM_CHARTER.md

```diff
@@ -0,0 +1,88 @@
+This document defines how the OpenJS Foundation implements its code of conduct.
+
+If you want to report an incident, go to [code-of-conduct.openjsf.org][CoC].
+
+***
+
+# Code of Conduct Team Charter
+
+The Cross Project Council (CPC) delegates enforcement of the [Code of Conduct][CoC] to the Code of Conduct Team (CoC Team) through this Code of Conduct Team Charter. This charter defines the composition and roles of the CoC Team, along with CoC team responsibilities and operations.
+
+## Responsibilities
+
+The CoC Team's responsibilities are to:
+
+- Handle code of conduct [incident reports][incident report].
+- Handle code of conduct [escalation requests][escalation].
+- Handle code of conduct [appeals][appeal].
+- Keep the CPC up to date of cases requiring the CPC's attention.
+- Provide regular transparency reports to the CPC.
+- Uphold this charter, the [code of conduct][CoC], and the [code of conduct policy][CoC Policy].
+
+## CoC Team Structure
+
+The CoC Team is composed of five (5) [primary members][#primary-members] and a variable number of [alternate community members][#alternate-community-members].
+
+### Primary members
+
+Primary members are CoC Team members responsible for [handling cases][#handling-cases].
+
+The five primary members are:
+
+* The OpenJS Foundation Executive Director
+* The OpenJS Foundation Communications Manager
+* Three (3) community members
+
+The name and email address of the primary members are documented in the [code of conduct][primary members].
+
+### Alternate community members
+
+Alternate community members are CoC Team Members that are available as backup when there aren't enough primary community members available to [handle a case][#handling-cases].
+
+### Member selection
+
+The Cross Project Council (CPC) will hold an annual selection process in order to select primary and alternate community members.
```

Comment:
presumably we'll hold this process (for the first time) ASAP once this PR lands?